dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/airflow/AirflowConnector.java

```diff
@@ -144,12 +265,45 @@ public void validate(ConnectorArguments arguments) throws IllegalStateException
       Preconditions.checkState(arguments.getPort() != null, "--port is required with --host");
       Preconditions.checkState(arguments.getSchema() != null, "--schema is required with --host");
     }
+
+    validateDatesInterval(arguments);
+  }
+
+  private void validateDatesInterval(ConnectorArguments arguments) {
```

Comment:
Shouldn't the validation be executed on higher level than the airflow connector (or at least function moved to some shared space)? New flags are not defined in the scope of the airflow connector and they could be used by other connectors too.